model/startDb.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"flag"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+func StartDB() (*gorm.DB, error) {
+
+	db_user := flag.String("user", "postgres", "database user")
+	db_password := flag.String("password", "MohanNeelima@01", "database password")
```

Comment:
Put such delicate details in .env files and fetch by os module

Reply:
created .env to store critical data